src/Microsoft.Azure.WebJobs.Host/Bindings/ILogger/ILoggerBinding.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Loggers;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    internal class ILoggerBinding : IBinding
```

Comment:
I didn't even notice that until you pointed it out -- it does make it look like an interface. I think I'll switch it to `LoggerBinding` -- that still conveys the right information, but doesn't look like an interface.